client/src/components/CenterpieceDropDown.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+
+const CenterpieceDropDown = (props) => {
+  console.log('this is friends prop: ', props.friends)
+  const clocks = props.friends.map((friend) => {
+      console.log('this is a friend in drop: ', friend)
+        return (
+          <Option key={friend} width="20px" value={friend} >{friend}</Option>
+          )
+  })
+
+  const changeClock = (e) => {
+    console.log('yeah buddy')
+    let userName = e.target.value
+    console.log('this is userName in DorpDown: ', userName)
+    axios.get('/api/users', {params:{userName}})
+    .then((res) => {
+    let oneUser = res.data;
+    console.log('this is oneUser: ', oneUser)
+    props.setLocations(oneUser.locations);
+    props.setCurrentLocation(oneUser.currentLocation)
+    props.setSuppress(true)
+  })
+  .catch(err => {throw err;})
```

Comment:
You probably shouldn't do this. See: https://stackoverflow.com/questions/30715367/why-can-i-not-throw-inside-a-promise-catch-handler